api_utils/txtlib.py

```
@@ -0,0 +1,131 @@
+    # ---
+    if lowers:
+        temps = [x.lower() for x in temps]
+    # ---
+    named = {}
+    # ---
+    if get_all_temps:
+        named = []
+    # ---
+    for temp in ingr:
+        # ---
+        name, namestrip, params, template = temp['name'], temp['namestrip'], temp['params'], temp['item']
```

Ruff (0.11.9)

Local variable `name` is assigned to but never used

Remove assignment to unused variable `name`

(F841)